tool/jt.rb

```diff
@@ -2043,14 +2047,19 @@ def graph(*args)
       "--engine.CompileOnly=#{method}",
       '--engine.MultiTier=false',
       '--engine.NodeSourcePositions',
+      '--vm.Dgraal.FullUnroll=false',
+      '--vm.Dgraal.PartialUnroll=false',
+      '--vm.Dgraal.LoopPeeling=false',
+      '--vm.Dgraal.LoopUnswitch=false',
+      '--vm.Dgraal.OptScheduleOutOfLoops=false',
       '--vm.Dgraal.PrintGraphWithSchedule=true',
       '--vm.Dgraal.PrintBackendCFG=true',
       '--vm.Dgraal.Dump=Truffle:1',
     ]
```

Comment:
I think we need to append `args` here or so, the logic to pass vm-args doesn't seem complete.

Reply:
Will move into a different PR.